addons/map/README.md

```diff
@@ -8,6 +8,10 @@ Various tweaks to the in-game map, including:
 - Map illumination (optional)
 - Blue Force Tracker (optional)
 
+#### Variables
```

Comment:
If I'm not mistaken line 12 has zero leading spaces and line 13 has one; at least that's how GitHub is showing it. Obviously it's a very minor thing, but it should be fixed if it's not a huge inconvenience.